tasks/prime.py

```diff
@@ -25,7 +25,7 @@ class PrimeTasks(CertTaskSet):
     tags where appropriate to make filtering for custom tests easier.
     """
 
-    @tag("mto")
+    @tag("mto", "fetchMTOUpdates")
```

Comment:
Are locust tags case-sensitive?

Reply:
I wasn't sure, but I just tested it and they are! We may end up deciding we want a different tagging scheme at some point, I think